CONTRIBUTING.md

```diff
@@ -106,11 +106,11 @@ I personally do not like something that requires so many configurations before y
 
 ## Tools
 
-- Node.js >= 14
-- NPM >= 8.5
-- Git
-- IDE that supports ESLint and EditorConfig (I am using IntelliJ IDEA)
-- A SQLite GUI tool (SQLite Expert Personal is suggested)
+- [`Node.js`](https://nodejs.org/de) >= 14
+- [`npm`](https://www.npmjs.com/) >= 8.5
+- [`git`](https://git-scm.com/)
+- IDE that supports [`ESLint`](https://eslint.org/) and EditorConfig (I am using [`IntelliJ IDEA`](https://www.jetbrains.com/de-de/idea/))
```

Comment:
Same here, should be `https://www.jetbrains.com/idea/`

Reply:
```suggestion
- IDE that supports [`ESLint`](https://eslint.org/) and EditorConfig (I am using [`IntelliJ IDEA`](https://www.jetbrains.com/idea/))
```